vyper/semantics/validation/annotation.py

```diff
@@ -132,6 +136,8 @@ def visit_Call(self, node, type_):
             # events and internal function calls
             for arg, arg_type in zip(node.args, list(call_type.arguments.values())):
                 self.visit(arg, arg_type)
+            for kwarg in node.keywords:
+                self.visit(kwarg.value, None)
```

Comment:
this line should probably annotate with the kwarg type no?

Reply:
I think this is related to this above thread about adding `infer_kwarg_types` for `ContractFunction`: https://github.com/vyperlang/vyper/pull/2817#discussion_r872490998